composer.json

```diff
@@ -34,7 +33,9 @@
     },
     "autoload": {
         "psr-4": {
-            "Kanvas\\Libraries\\": "src/"
+            "Kanvas\\Libraries\\": "src/",
```

Comment:
@dasdo Kanvas\Package Plain should be the and that it you dont need any of the other folder since this is PSR4